src/sensai/featuregen.py

```diff
@@ -194,16 +216,21 @@ class RuleBasedFeatureGenerator(FeatureGenerator, ABC):
     def fit(self, X, Y=None, ctx=None):
         pass
 
+    def _fit(self, X: pd.DataFrame, Y: pd.DataFrame = None, ctx=None):
+        pass
+
+    def isFitted(self):
+        return True
+
 
 class MultiFeatureGenerator(FeatureGenerator):
     """
     Wrapper for multiple feature generators. Calling generate here applies all given feature generators independently and
     returns the concatenation of their outputs
     """
-    def __init__(self, featureGenerators: Sequence[FeatureGenerator]):
-        """
-        :param featureGenerators:
-        """
+    def __init__(self, *featureGenerators: FeatureGenerator):
```

Comment:
This is not an acceptable breaking change.

Reply:
Will be resolved by using the flattening mechanism everywhere for now

Reply:
Perhaps add the flatten function to a util module?

Reply:
Definitely!

Reply:
Done